communication/moderators.md

```diff
@@ -136,6 +136,7 @@ Moderators seats: 9
 - Paris Pittman (@parispittman) - PT
 - Jorge Alarcon (@alejandrox1) - ET
 - Yang Li (@idealhack) - CST
+- Noah Kantrowitz (@coderanger) - America/Los_Angeles
```

Comment:
Can we all use UTC? :stuck_out_tongue:

Reply:
Oh wait...UTC doesn't observe DST.

I'll leave this to our timezone experts. :woman_shrugging:

Reply:
@coderanger Can you please fix the timezone to match everyone else? :D

PS. I gonna make the TZ's more readable with https://github.com/kubernetes/community/issues/3380, but for now please do it in an abbreviation format.